static/scss/answers/interactive-map/VerticalFullPageMap.scss

```diff
@@ -562,9 +562,8 @@
     }
 
     &-closeCardButtonWrapper {
-      position: absolute;
```

Comment:
is there some way we could add a max-width HitchhikerLocationStandard-title instead? if not I'm happy with the current solution

Reply:
Ya, we could add a max width, however we would need to account for the size of the distance text which can vary. It will change based on the distance and it would change if someone customized the font or changed the text size. 

We would also need to modify the size of the card content because that is also at risk of overflowing. That'a a lot trickier because we don't have guarantees about the layout of the content (it will change based on the location card). We could add a mix width to the whole content div as well, however that would prevent the content from filling the whole card.

Reply:
does changing the closeCardButton to position: relative take care of the card content overflowing? I don't seem to be able to get the card content to overflow but I could be gumming something up.

I was thinking realistically the distance text will only be at most something like "20,000km" (7 characters + comma), since the earth's circumference is 40,000km (maybe slightly longer if some locale's number format has extra commas/spaces. Do you think something like a max width of 75% on the title and 25% on the closeCardWrapper could work? Or maybe a fixed max-width on the closeCardWrapper and a `100% - {width}` on the title? I know I'm being kind of persistent about this, but I think most of the time the title won't overflow the card. If possible I'd rather optimize our default css for the title not overflowing, than vice versa

Reply:
Setting closeCardButton to position: relative didn't seem to take care of the card content overflowing:

![Screen Shot 2021-04-12 at 11 50 19 AM](https://user-images.githubusercontent.com/42848445/114423894-9e140580-9b85-11eb-8a44-07a237a376a5.png)

I tried out your suggested title fixes, however the card body can still overflow if there is a long address or a large body of text in the description right at the top of the card. Rose said that she'd rather have some extra spacing than having the text overflow. And she mentioned that a lot of healthcare entities have really long titles, so long titles do occur pretty frequently.

I think that if we want to avoid the extra spacing, we should change our layout so that the location distance is to the left of the close card button. However, Rose didn't want to make such a layout change at this point in time.

While testing out the other cards, I noticed that I need to make a small flexbox change, so I will update that now

Reply:
ah ok, it sucks we can't get that extra real estate on mobile but it sounds like we kind of have an impossible (or very difficult) css problem

Reply:
Yep, I think in the future we want to be careful with `position: absolute` because it can cause these sorts of overflow issues. I would also like to fix the spacing, and I think that moving the distance to the left of the close button is the right way to do it. I'll make an item for it